tests/run.bats

```diff
@@ -1498,4 +1502,43 @@ export BUILDKITE_JOB_ID=1111
   assert_output --partial "ran myservice without tty"
   unstub docker-compose
   unstub buildkite-agent
-}
\ No newline at end of file
+}
+
+@test "Run with Docker labels" {
+  export BUILDKITE_JOB_ID=1111
+  export BUILDKITE_PLUGIN_DOCKER_COMPOSE_RUN=myservice
+  export BUILDKITE_PIPELINE_SLUG=test
+  export BUILDKITE_BUILD_NUMBER=1
+  export BUILDKITE_COMMAND=pwd
+  export BUILDKITE_PLUGIN_DOCKER_COMPOSE_RUN_LABELS="true"
+  export BUILDKITE_PIPELINE_NAME="label-test"
+  export BUILDKITE_LABEL="Testjob"
+  export BUILDKITE_STEP_KEY="test-job"
+  export BUILDKITE_AGENT_NAME="agent"
+  export BUILDKITE_AGENT_ID="1234"
```

Comment:
If I'm being quite picky, I'd rather these variables be split in two groups: those general to the pipeline and those that are specific to the plugin. And those groups be alphabetically sorted while we are at it :P (feel free to ignore this completely cosmetic and personal opinion)

Reply:
updated